.gitignore

```diff
@@ -1,58 +1,59 @@
-*.class
-src/hiscore.txt
-
-# IntelliJ stuff
-.idea/
-*.iml
-out/
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.ear
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-
-# =========================
-# Operating System Files
-# =========================
-
-# OSX
-# =========================
-
+*.class
+src/client/hiscore.txt
```

Comment:
Remove this, as we no longer store high score

Reply:
Fixed